packages/community-jazz-vue/src/inspector/JazzInspector.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <VueJazzInspector :position="position" />
+</template>
```

Comment:
We have created the [custom element version](https://jazz.tools/docs/vanilla/inspector#embedding-the-inspector-widget-into-your-app) of the Inspector to be used by all the non-React frameworks.

The goal was to keep maintaining only one inspector while still supporting all the frameworks.

How about moving to a Vue wrapper component around [the custom element](https://jazz.tools/docs/vanilla/inspector#embedding-the-inspector-widget-into-your-app) provided by jazz-tools?